src/allotropy/parsers/agilent_gen5_image/agilent_gen5_image_structure.py

```diff
@@ -350,6 +351,7 @@ def _create_measurement(
         detector_gain_setting=instrument_settings.detector_gain,
         magnification_setting=read_section.magnification_setting,
         illumination_setting=instrument_settings.illumination,
+        illumination_setting_unit=UNITLESS,
```

Comment:
NOTE: illumination can be unitless or %, revvity uses %, so we specify unit now.